cfme/fixtures/targeted_refresh.py

```diff
@@ -0,0 +1,32 @@
+"""
+The idea of this fixture is to pass some entity ids to
+:py:class:`utils.targeted_refresh.TargetedRefresh` and check the evm.log
+whether all expected targeted refreshes
+were triggered.
+
+register_targeted_refresh accepts attributes for entity that is expected to trigger refresh
+"""
+import pytest
+from cfme.utils.targeted_refresh import TargetedRefresh
+
+
+@pytest.fixture(scope="function")
+def register_targeted_refresh(soft_assert, merkyl_inspector):
```

Comment:
Can we change fixture name to `targeted_refresh` and make it return object with method `register_target`? I would find it more readable in test to say:

```
targeted_refresh.register_target(...)
targeted_refresh.register_target(...)
targeted_refresh.register_target(...)
```

There is also a comment in register_event suggesting the same, saying literally `bad idea, to replace by object later`, see https://github.com/ManageIQ/integration_tests/blob/master/cfme/utils/events.py#L283